### ISSUE TITLE
New PyPI release and/or instructions to build from source

### ISSUE THREAD

**Issue by chisarie** [owl-project/NVISII#165]
Hello, 

First of all, thank you very much for this amazing project! It looks incredible how easy to use it is, given the high quality renderings it produces.

I noticed that the latest release on PyPI is almost 2 years old now, even though quite some development happened since then, in both this repo and the owl one. Would it be possible to add a new release on PyPI and/or instructions on how to build the project from source ourselves?

Thank you very much in advance!

**Comment by SeanNobel** [comment]
Yes, would be highly appreciated to have a python=3.10 build. Thank you for the amazing work.

**Comment by hiyyg** [comment]
> Yes, would be highly appreciated to have a python=3.10 build. Thank you for the amazing work.

Same request for python=3.10 build support!
